### PR TITLE
[ISSUE #26] Make interface orthogonal

### PR DESCRIPTION
**Pull request by drpmma**

### CODE REVIEW

**Comment by aaron-ai** [comment]
LGTM.

File: apache/rocketmq/v2/service.proto [hunk 1]
```diff
@@ -80,6 +84,8 @@ message ReceiveMessageRequest {
   int32 batch_size = 4;
   // Required if client type is simple consumer.
   optional google.protobuf.Duration invisible_duration = 5;
+  // For message auto renew and clean
+  bool auto_renew = 6;
```
Comment (lizhanhui) [inline apache/rocketmq/v2/service.proto:88]: What is the rationale for this change?

Comment (drpmma) [reply apache/rocketmq/v2/service.proto:88]: The field is used to distinguish messages' renew and clean logic. And the server will know whether to manage messages by request other than looking up in `Settings`.

**Review by zhouxinyu — APPROVED** [review]

File: apache/rocketmq/v2/service.proto (hunk 1 above)
Comment (lizhanhui) [reply apache/rocketmq/v2/service.proto:88]: If I am guessing correctly, you are assuming this field is a flag instructing proxies to extend the invisible duration of messages on behalf of clients. 

It would be kind of strange adding this flag to this RPC. Would it be better to add this flag to the receive-message-RPC?
